kubetest/e2e.go

```diff
@@ -762,18 +762,24 @@ func kubemarkGinkgoTest(testArgs []string, dump string) error {
 }
 
 // Brings down the kubemark cluster.
-func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump string) {
+func kubemarkDown(err *error, wg *sync.WaitGroup, provider, dump string, logexporterGCSPath string) {
 	defer wg.Done()
 	control.XMLWrap(&suite, "Kubemark MasterLogDump", func() error {
 		logDumpPath := logDumpPath(provider)
-		cmd := exec.Command(logDumpPath, dump)
 		masterName := os.Getenv("MASTER_NAME")
+		var cmd *exec.Cmd
 		cmd.Env = append(
 			os.Environ(),
 			"KUBEMARK_MASTER_NAME="+masterName,
 			"DUMP_ONLY_MASTER_LOGS=true",
 		)
-		log.Printf("Dumping logs for kubemark master: %s", masterName)
+		if logexporterGCSPath != ""{
+			log.Printf("Dumping logs for kubemark master to GCS directly at path: %v", logexporterGCSPath)
+			cmd = exec.Command(logDumpPath, dump, logexporterGCSPath)
+		}else{
```

Comment:
Please add a spaces between "else" and "{"/"}" to make it "} else {" to follow style used in this file.